scripts/globals/items/guttler.lua

```diff
@@ -10,12 +10,13 @@ require("scripts/globals/magic");
 -- onAdditionalEffect Action
 -----------------------------------
 function onAdditionalEffect(player,target,damage)
+    
     local chance = 10;
-
+    
     if (math.random(0,99) >= chance or applyResistanceAddEffect(player,target,ELE_ICE,0) <= 0.5) then
         return 0,0,0;
-    else
-        target:addStatusEffect(EFFECT_CHOKE, 17, 0, 60);
-        return SUBEFFECT_CHOKE, MSGBASIC_ADD_EFFECT_STATUS, EFFECT_CHOKE;

```

Comment:
That makes no sense. Either the condition is triggered or not, there is no 3rd possibility. All the other scripts use the previous format. Its a change with zero purpose.